packages/cli/src/util/link/setup-and-link.ts

```diff
@@ -80,6 +80,7 @@ export default async function setupAndLink(
   const shouldStartSetup =
```

Comment:
Line 59: Should this `const isTTY = process.stdout.isTTY;` check `client.stdout.isTTY` instead?